adidas-ascii-logo/README.md

```diff
@@ -0,0 +1,54 @@
+# adidas ASCII logo
+
+This challenge consists of implementing a function which, with the width of the stripe, returns the adidas logo using `@` character.
+
+_Extra:_ the goal is not only to return the adidas logo, but implementing the function with the less number of characters possible.
+
+## Requirements
+
+It has to satisfy the next requirements:
+
+- The function accepts a number which is the width of on of the three adidas brand logo stripes.
+   - The minimun width is `2`.
+   - With less than `2` of width, the function throws the exception: `Error, minimun width is 2`.
+- The function returns a string with at signs, whitespaces and new line characters.
```

Comment:
`[...] with at signs, whitespaces and new line characters` -> ``with "at" (`@`), whitespace (` `) and new line (`\n`) characters``

And then we can remove next line